pkg/oauth/github/github.go

```diff
@@ -104,7 +104,7 @@ func (c *OAuthClient) GetUser(ctx context.Context) (*model.User, error) {
 	if err != nil {
 		return nil, err
 	}
-	teams, _, err := c.Teams.ListUserTeams(ctx, nil)
+	teams, _, err := c.Teams.ListUserTeams(ctx, &github.ListOptions{PerPage: 100})
```

Comment:
this `100` value should be defined at this file level as const to avoid magic number pattern.

Reply:
Thanks for the review.
I have corrected it.